changelog/dwarf-switch.dd

```diff
@@ -0,0 +1,9 @@
+Add `-gdwarf-<version>` compiler switch
```

Comment:
Shouldn't the switch be -gdwarf=5 to be consistent with DMD's other flags? Then, it's also possible for the user to list all available options.

Reply:
Maybe, but Clang and GCC have the `-gdwarf-<version>` switch, so I added the same for consistency between different compilers. What do you think ?

Reply:
I think sticking to the DMD style makes more sense here, because I am not sure a lot of people would like to have at least three more CLI flags. IIRC there has been a precedent with make flags. What do other people think?

Reply:
On the forums, people are unanimous. Moving to DMD style.